### PR TITLE
Scrape vim-plug managed plugins

### PR DESCRIPTION
**Pull request by morhetz**
Fixes issue #41

### CODE REVIEW

**Comment by divad12** [comment]
Sweet! This looks like it'll work. Can you comment briefly on any testing you did?

File: tools/scrape/github.py [hunk 1]
```diff
@@ -472,15 +474,21 @@ def _extract_bundle_repos_from_file(file_contents):
         file_contents: A string of the contents of the file to search through.
 
     Returns:
-        A tuple (Vundle repos, NeoBundle repos). Each element is a list of
-        tuples of the form (owner, repo_name) referencing a GitHub repo.
+        A tuple (Vundle repos, NeoBundle, VimPlug repos). Each element is a list
+        of tuples of the form (owner, repo_name) referencing a GitHub repo.
```
Comment (divad12) [inline tools/scrape/github.py]: We're not returning a tuple here... you changed this to return a dict. Mind updating the docstring here accordingly?

**Comment by captbaritone** [comment]
@morhetz Any interest in incorporating @divad12's feedback? If not I'd be happy to cherry-pick your commit and incorporate the feedback on your behalf.

**Comment by morhetz** [comment]
@captbaritone Please do!
I really have neither free time no desire to comment the stuff
Thanks

**Comment by captbaritone** [comment]
:+1:

**Comment by divad12** [comment]
Thanks @captbaritone

**Comment by captbaritone** [comment]
Closed in favor of #81.

**Pull request closed without merging**